runtime/calamari/src/xcm_config.rs

```diff
@@ -180,6 +181,24 @@ parameter_types! {
     pub XcmFeesAccount: AccountId = Treasury::account_id();
 }
 
+/// Xcm fee of native token
+pub struct XcmNativeFeeToTreasury;
```

Comment:
Seems `XcmNativeFeeToTreasury` is not added to manta, not sure why.

Reply:
Manta don't have treasury account yet. We can have a future PR for manta.